_layouts/blog.html

```diff
@@ -1,16 +1,22 @@
 ---
-layout: default
+layout: page
```

Comment:
Okay. Then all I have design wise for the other pages is to add a color background to the header bar for other pages which don't use the splash image. For example I like how Krita's website pull this off between its [home page](https://krita.org) the [other pages](https://krita.org/en/about/history/) So at least the page flow doesn't break.